src/styles/utiles/variables.scss

```
@@ -0,0 +1,14 @@
+$calendar__day-height: 100px;
+$calendar__day-width: 100px;
```

Don't need to duplicate variables with the same values
```suggestion
$day-size: 100px;
```